client-cli/src/command/address_command.rs

```diff
@@ -139,7 +139,11 @@ impl AddressCommand {
 
         let pub_keys = match address_type {
             AddressType::Staking => wallet_client.staking_keys(name, &enckey)?,
-            AddressType::Transfer => wallet_client.public_keys(name, &enckey)?,
+            AddressType::Transfer => wallet_client
+                .public_keys(name, &enckey)?
+                .iter()
```

Comment:
Why not use `.into_iter()` instead of `.iter().cloned()`?

Reply:
Good catch, fixed.